cmd/server/app/migrate_up.go

```diff
@@ -152,6 +160,43 @@ func ensureAuthzStore(ctx context.Context, cmd *cobra.Command, authzw *authz.Cli
 	return storeID, nil
 }
 
+func migratePermsToFGA(ctx context.Context, store db.Store, authzw *authz.ClientWrapper, cmd *cobra.Command) error {
+	cmd.Println("Migrating permissions to FGA...")
+
+	userlist, err := store.ListUsers(ctx, db.ListUsersParams{
+		Limit:  1000,
+		Offset: 0,
+	})
```

Comment:
I'm assuming the limit here is something that can be changed if we need to run this and we have 10k users / projects?

Reply:
Alternative would be to use a `for` loop:

```golang
i := 0
for {
  userList, err := store.ListUsers(ctx, db.ListUsersParams{Limit: 100, Offset: i})
  if err != nil {
    // report err, exit
  }
  if len(userList) == 0 {
    break
  }
  for _, user := range userlist {
    // ...
  }
}
```